sqla_yaml_fixtures.py

```diff
@@ -1,13 +1,19 @@
 __version__ = (0, 1, 2)
 
-import functools
+try:
+    # Python 3
+    from functools import lru_cache
+except ImportError:
+    # For Python 2: 
+    # use https://pypi.python.org/pypi/backports.functools_lru_cache
+    # pip install backports.functools_lru_cache
+    from backports.functools_lru_cache import lru_cache
 
 import yaml
 import sqlalchemy
 from sqlalchemy.orm.relationships import RelationshipProperty
 
 
-
```

Comment:
ok, I use `flake8` in other projects in CI. Thought it was included If I run it I get this warning and many others:

```
./sqla_yaml_fixtures.py:7:20: W291 trailing whitespace
./sqla_yaml_fixtures.py:12:1: E402 module level import not at top of file
./sqla_yaml_fixtures.py:13:1: E402 module level import not at top of file
./sqla_yaml_fixtures.py:14:1: E402 module level import not at top of file
./sqla_yaml_fixtures.py:44:5: E265 block comment should start with '# '
./sqla_yaml_fixtures.py:50:11: E271 multiple spaces after keyword
./sqla_yaml_fixtures.py:56:1: E303 too many blank lines (3)
./sqla_yaml_fixtures.py:72:14: E261 at least two spaces before inline comment
./sqla_yaml_fixtures.py:99:25: E261 at least two spaces before inline comment
./test_sqla_yaml_fixtures.py:16:1: E302 expected 2 blank lines, found 1
./test_sqla_yaml_fixtures.py:33:80: E501 line too long (80 > 79 characters)
./test_sqla_yaml_fixtures.py:71:1: E303 too many blank lines (3)
./test_sqla_yaml_fixtures.py:75:1: E302 expected 2 blank lines, found 3
./test_sqla_yaml_fixtures.py:79:1: E302 expected 2 blank lines, found 1
./test_sqla_yaml_fixtures.py:85:1: E302 expected 2 blank lines, found 1
./test_sqla_yaml_fixtures.py:130:1: E303 too many blank lines (3)
./test_sqla_yaml_fixtures.py:209:1: E303 too many blank lines (3)
./test_sqla_yaml_fixtures.py:260:1: W391 blank line at end of file

```